src/components/tile/tile-footer/tile-footer.component.js

```diff
@@ -18,8 +18,8 @@ const TileFooter = ({ variant, children, ...props }) => {
 
 TileFooter.propTypes = {
   ...paddingPropTypes,
-  /** set which background color variant should be used */
-  variant: PropTypes.oneOf(["default", "transparent"]),
+  /** Sets which background color variant should be used */
+  variant: PropTypes.oneOf(["default", "black", "transparent"]),
```

Comment:
Similarly we should also update the types in `tile-footer.d.ts` to reflect this change

Reply:
Done.